astroid/nodes/node_ng.py

```diff
@@ -448,7 +445,7 @@ def _fixed_source_line(self) -> Optional[int]:
         We need this method since not all nodes have :attr:`lineno` set.
         """
         line = self.lineno
-        _node: Optional[NodeNG] = self
+        _node: Optional[NodeNG] = self  # pylint: disable = used-before-assignment
```

Comment:
Is this a new false positive?

Reply:
https://github.com/PyCQA/pylint/issues/5342

Reply:
Ah! Thanks!